test/app-luatest/msgpack_test.lua

```diff
@@ -219,11 +219,11 @@ g.test_object_encode_decode = function()
     t.assert_equals(
         msgpack.decode(msgpack.encode({mp, {foo = mp}})),
         {mp:decode(), {foo = mp:decode()}})
-    t.assert_equals(msgpack.object(box.tuple.new(1, 2, 3)):decode(), {1, 2, 3})
+    t.assert_equals(msgpack.object(box.tuple.new{1, 2, 3}):decode(), {1, 2, 3})
     t.assert_equals(
         msgpack.object({
-            foo = box.tuple.new(123),
-            bar = box.tuple.new(456),
+            foo = box.tuple.new{123},
+            bar = box.tuple.new{456},
```

Comment:
I would revert the test changes, where just one scalar is passed. It is documented and should work.

Reply:
Left a comment above about forbidding scalars.

Reply:
Fixed.

Reply:
This was unintended, sorry, I forgot to revert some tests: fixed, now there should be no unnecessary changes.